recipe/meta.yaml

```diff
@@ -242,11 +237,11 @@ outputs:
       - include/3rdparty
       - include/fmt
       - include/gxf
-      - include/gxf_holoscan_wrapper
       - include/holoinfer
       - include/holoscan
       - include/holoviz
       - include/magic_enum.hpp
+      - include/rapids_logger
```

Comment:
@bdice, can we move the rapids-logger package to conda-forge, so that holoscan doesn't redistribute it?

Reply:
https://anaconda.org/channels/rapidsai/packages/rapids-logger/overview

Reply:
I have no objection to trying to publish rapids-logger to conda-forge. The rest of RAPIDS would be a heavier lift for a variety of reasons though. We archived the rmm package (which @carterbox [asked about below](https://github.com/conda-forge/holoscan-feedstock/pull/17#discussion_r2586772839)) because dual-publishing is more work and is particularly error-prone when you cannot simply redistribute binary artifacts. We're not in a position to change RAPIDS's publishing any time soon.